crypto/tls/tls.go

```diff
@@ -72,10 +72,17 @@ func (cfg *ClientConfig) GetTLSConfig() (*tls.Config, error) {
 	return config, nil
 }
 
-// GetGRPCDialOptions creates GRPC DialOptions for TLS
+// WithInsecure wraps grpc.WithInsecure.
+//
+// Stubbable for tests.
+var WithInsecure = func() grpc.DialOption {
```

Comment:
Do we wanna to expose it? Looks weird to me.

Reply:
It's technically necessary for grpcclient tests to be able to stub calling out to the `grpc` package. I'll look again into whether there's another way to do this (dependency injection?).